src/components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/V2V3PayForm.tsx

```diff
@@ -64,8 +64,10 @@ export const V2V3PayForm = ({
     const initialStickerUrls = nftRewardTiers?.map(
       (tier: NftRewardTier) => tier.imageUrl,
     )
+
     form.setFieldsValue({ stickerUrls: initialStickerUrls })
-  })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
**question** is there a way that we can not ahve this line? Maybe we need `form` as dependency.

Handing this over to @wraeth-eth for reivew though <3

Reply:
I've seen some discussion about the benefits of not disabling the exhaustive deps warning in the useEffect hook, it is recommended in most cases to not disable it and I see a reason for that.

I'm still not sure why we would need to include the form object in the dependencies array here, since we're only setting the form value when nftRewardTiers changes. I'm happy to discuss this more and hear your thoughts! 🍻